bootstrap-scripts/service_to_role.json

```diff
@@ -8,5 +8,6 @@
     "kafka-manager": {"role": "kafka_manager", "port": "10900"},
     "logserver": {"role": "logserver", "port": "5601"},
     "opentsdb": {"role": "opentsdb", "port": "4242"},
-    "knox": {"role": "knox", "port": "8443"}
-}
+    "knox": {"role": "knox", "port": "8443"},
+    "haproxy": {"role": "haproxy", "port": "8443"}
```

Comment:
Given haproxy is deployed on the same node as the gateway, it won't be the same port